.circleci/config.yml

```diff
@@ -0,0 +1,29 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/elixir:1.5.3
+    working_directory: /home/circleci/formulator
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - dependency-cache-{{ arch }}-{{ .Branch }}-{{ checksum "mix.lock" }}
+          - dependency-cache-{{ arch }}-{{ .Branch }}
+          - dependency-cache-{{ arch }}
+      - run: mix local.hex --force && mix local.rebar --force && mix deps.get
```

Comment:
Why not `bin/setup`? Just curious.

Reply:
There is  more setup needed on a dev machine than on CI. I think it makes sense to leave `bin/setup` as local dev setup and CI can handle it's own thing.

If it gets more crazy than just these commands, we'll probably need a `bin/ci_setup`